fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1858,22 +1858,62 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 	void set(KeyValueRef kv, const Arena*) override {
 		if (writeBatch == nullptr) {
 			writeBatch.reset(new rocksdb::WriteBatch());
+			setKeysSet.clear();
```

Comment:
Should we do a return here after the clear? We keep going even if null .. .and then a deference on #1863

Reply:
Talked to Michael and it looks good.